TeamCode/src/main/java/com/acmerobotics/relicrecovery/vision/CryptoboxTracker.java

```diff
@@ -62,16 +58,18 @@
 public class CryptoboxTracker implements Tracker {
     public static final String TAG = "CryptoboxTracker";
 
+    // TODO: hack!
+    public static boolean isUnitTest;
```

Comment:
@kellyrm @mabeeenigma I really want a better solution to this. The main problem is that this same class is used by unit tests (no Android/non-OpenCV native code). Normally, this would be fine; I would just mock out the Android/Vuforia classes. In this case, we need to get a `CameraDevice` object to find the camera's FOV. Unfortunately, this code is native Vuforia and can only be obtained at runtime using `CameraDevice.getInstance()` inside this class, making it difficult to mock (Mockito doesn't like static methods).

Reply:
I agree that it is not ideal, but I also don't think it is that terrible. :) You have a clear and valid reason for breaking the normal pattern in pursuit of overall better quality code (i.e. code that has tests). I also don't currently have a better idea. I would suggest considering expanding the comment so that you future self, or Kelly next year, understands why you did what you did. Your explanation from above would be fine.